mithril-aggregator/src/configuration.rs

```diff
@@ -395,6 +399,9 @@ pub struct DefaultConfiguration {
 
     /// Metrics HTTP server listening port.
     pub metrics_server_port: u16,
+
+    /// Time interval at which metrics are persisted in event database (in seconds).
+    pub persist_usage_report_interval_in_seconds: u64,
```

Comment:
Can you update the [aggregator documentation](https://mithril.network/doc/manual/developer-docs/nodes/mithril-aggregator#subcommands) with this new parameter?

Reply:
Done